src/main/kotlin/com/avail/interpreter/primitive/sockets/P_SocketIPv6Connect.kt

```diff
@@ -74,7 +80,7 @@ import java.util.Collections.emptyList
 
 /**
  * **Primitive:** Connect the [AsynchronousSocketChannel] referenced by the
- * specified [handle][A_Atom] to an [IPv4&#32;address][Inet4Address] and
+ * specified [handle][A_Atom] to an [IPv4 address][Inet4Address] and
```

Comment:
`&#32;`

Reply:
This comment should also be able `IPv6Address`. I fixed that too.